### ISSUE TITLE
Enter one-line summary

### ISSUE THREAD

**Issue by GoogleCodeExporter** [evancharlton/android-mileage#10]
```
What steps will reproduce the problem?
1. Have 2 vehicule
2. Put your phone in vertical position and select the second vehicle
3. change the orientation of your phone ( put it horizontal )

Expected behavior: The vehicle didn't change

Actual behavior: The first vehicle is selected when orientation change

Mileage version: 2.2.6
Android version: 2.2.1
Device: Htc desire Z
```

Original issue reported on code.google.com by `fr...@starox.org` on 29 Jan 2011 at 9:16

**Comment by GoogleCodeExporter** [comment]
```
Sorry I forgot to give a title ...
```

Original comment by `fr...@starox.org` on 29 Jan 2011 at 9:20

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Mileage 3.0
```

Original comment by `evancharlton` on 3 Mar 2011 at 1:38
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `e...@evancharlton.com` on 22 Apr 2012 at 3:44
- Changed state: **Released**
